### DISCUSSION TITLE
cache-eviction options

### DISCUSSION THREAD

**Discussion by imtipi** [juicedata/juicefs#5381] [Q&A]
eng:
I see that juicefs's cache-eviction only has 2-random and none, does this mean that LRU or LFU will not be used for eviction when the cache space is full, and will this affect the cache hit rate?

chs:
我看见juicefs的cache-eviction只有2-random和none，请问这是否代表着缓存空间填满时不会使用LRU或LFU进行驱逐，这是否会影响缓存的hitrate？

**Comment by zhijian-pro** [comment]
未满之前不会驱逐，不会影响缓存的命中率

**Reply by imtipi** [reply]
但我想问的就是在已满的时候，驱逐是使用的什么算法，是否可以选择lru或lfu？

**Reply by zhijian-pro** [reply]
默认是2-random 不可选择

**Reply by hulexwang** [reply]
可以选择n random算法，减少误删除的可能性

**Reply by solracsf** [reply]
A good read: https://sking7.github.io/articles/755965715.html
